Toolset/palettes/tutorial/courses/First Run/tutorials/BMI Calculator/lessons/App Lesson.txt

```diff
@@ -786,7 +825,8 @@ skip point
 
 step "Create Chart Card"
 	We are going to add a new card to the stack, to display a chart of 
-	BMI index data. Create a new card.
+	BMI index data. Create a new card by selecting Object > New Card 
```

Comment:
Should single quotes be used here, for consistency: 'Object' > 'New Card'?.  I wonder if it would improve the appearance to use `→` instead of `>` for menu paths?